compiler/ilgen/IlBuilder.cpp

```diff
@@ -770,6 +770,34 @@ IlBuilder::VectorLoadAt(TR::IlType *dt, TR::IlValue *address)
    return returnValue;
    }
 
+/**
+ * @brief IlBuilder::StructFieldAddress IlGen to get the address of a struct field
+ * @param structName is the name of the struct type the field belongs to
+ * @param fieldName is the name of the field itself
+ * @param obj is an IlValue representing the instance (address) of the struct
+ * @return IlValue representing the address of the field in the struct
+ */
+TR::IlValue *
+IlBuilder::StructFieldAddress(const char* structName, const char* fieldName, TR::IlValue* obj) {
+   auto offset = typeDictionary()->OffsetOf(structName, fieldName);
+   auto ptype = typeDictionary()->PointerTo(typeDictionary()->GetFieldType(structName, fieldName));
+   auto addr = Add(obj, ConstInt64(offset));
```

Comment:
@Leonardo2718 , please document the fact that this change is less than ideal, but it is a reasonable step forward for JitBuilder.  It may be revisited in the future as part of many other JitBuilder enhancements.  When @mstoodle returns we can review the change with him to get his opinion.  Vijay will merge after the comment is added.